src/overlays/actors/ovl_En_Md/z_en_md.h

```diff
@@ -6,9 +6,26 @@
 
 struct EnMd;
 
+typedef void (*EnMdActionFunc)(struct EnMd*, GlobalContext*);
+
 typedef struct EnMd {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1D8];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ EnMdActionFunc actionFunc;
+    /* 0x0194 */ ColliderCylinder collider;
+    /* 0x01E0 */ struct_80034A14_arg1 unk_1E0;
+    /* 0x0208 */ u8 unk_208;
+    /* 0x0209 */ u8 unk_209;
+    /* 0x020A */ u8 unk_20A;
+    /* 0x020B */ u8 unk_20B;
+    /* 0x020C */ s16 blinkCountdown;
+    /* 0x020E */ s16 eyeImageIdx;
+    /* 0x0210 */ s16 alpha; // s16 needed to match
```

Comment:
s16 is a fine type for an alpha value so there is no need to justify it, unless I'm missing something.
```suggestion
    /* 0x0210 */ s16 alpha;
```